cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -93,16 +92,16 @@ def get_distributed_appliances():
     return (appl1, appl2)
 
 
-def configure_db_replication(db_address):
+def configure_db_replication(db_address, appliance):
```

Comment:
This is not a fixture, and so this will fail, the lines that use this need to also pass in the appliance object